public/src/modules/helpers.js

```diff
@@ -117,7 +118,7 @@
 		}
 		category.children.forEach(function (child) {
 			if (child && !child.isSection) {
-				var link = child.link ? child.link : (relative_path + '/category/' + child.slug);
+				var link = child.link ? child.link : (config.relative_path + '/category/' + child.slug);
```

Comment:
I think this is causing tests to  fail somehow

Reply:
Yeah looks like `config.relative_path` is `undefined` in this file?